packages/fiori/src/themes/ViewSettingsDialog.css

```diff
@@ -12,13 +12,31 @@
 }
 
 .ui5-vsd-title {
-	margin-bottom: 0.3rem;
+	font-size: 1rem;
+	margin-left: 0;
+}
+
+.ui5-vsd-sub-title {
+	margin-left: 0;
+}
+
+.ui5-vsd-bar-sub-header {
+	padding-left: 0.5rem;
```

Comment:
same as the comment below.

Reply:
I meant the comment below in the same file - line 29, not the one below in sequence.

Reply:
done